### ISSUE TITLE
Issue with the intermediate results for long utterances

### ISSUE THREAD

**Issue by jin1004** [alumae/kaldi-gstreamer-server#85]
When I try to decode long utterances and I can see the intermediate results on the screen, sometimes several lines suddenly pop up on the screen with words that were decoded previously. These random lines in the middle are never present in the final results. Do you have any insight on why that tends to happen?

**Comment by alumae** [comment]
No, I have no explanation for that and I have not seen this myself.

**Comment by jin1004** [comment]
I have provided an example of this issue here. I got Result 1 and Result 2 (as shown below) back to back.

As mentioned earlier, most of the time results are normal like the one shown in Result2.png. But this strange thing happens with intermediate results every once in a while. 

It doesn't have any effect of the final result, but the system seems faulty when used for an online dictation type task. Any help regarding the matter will be greatly appreciated.

![result 1](https://user-images.githubusercontent.com/28697238/29200216-f4e6767a-7e85-11e7-8ade-f609f7feb7d5.png)
![result 2](https://user-images.githubusercontent.com/28697238/29200217-f6188dda-7e85-11e7-866c-5c970a481a71.png)

**Comment by lapwing** [comment]
I have exactly the same problem!
